plugins/jobs/src/js/components/JobStopRunModal.js

```diff
@@ -26,7 +29,11 @@ class JobStopRunModal extends React.Component {
       bodyText = "the selected job runs";
     }
 
-    return <span key="confirmText">You are about to stop {bodyText}.</span>;
+    return (
+      <span key="confirmText">
+        <Trans render="span">You are about to stop {bodyText}.</Trans>
```

Comment:
instead of using `bodyText` here should we have two seperate returns so the extracted messages are more complete sentences and easier to translate?